File: javascript/exercicios/ex003_par_impar.js
// Exercício 003 - Par ou Ímpar
// Objetivo: Praticar condicionais e operador módulo

function parOuImpar(numero) {
    if (numero % 2 === 0) {
        return `${numero} é PAR`;
    } else {
        return `${numero} é ÍMPAR`;
    }
}

console.log(parOuImpar(4));
console.log(parOuImpar(7));

File: c-cpp/exercicios/ex003_par_impar.c
// Exercício 003 - Par ou Ímpar
// Objetivo: Praticar condicionais
#include <stdio.h>

int main() {
    int numero;

    printf("Digite um número: ");
    scanf("%d", &numero);

    if (numero % 2 == 0) {
        printf("%d é PAR\n", numero);
    } else {
        printf("%d é ÍMPAR\n", numero);
    }

    return 0;
}
